frontend/src/pages/BYONImages/BYONImagesTable.tsx

```diff
@@ -330,6 +331,17 @@ export const BYONImagesTable: React.FC<BYONImagesTableProps> = ({ images, forceU
                       alignItems={{ default: 'alignItemsCenter' }}
                     >
                       <FlexItem>{image.name}</FlexItem>
+                      <FlexItem>
+                        <ResourceNameTooltip
+                          resource={{
+                            apiVersion: '',
```

Comment:
This looks very weird -- Can you not just use what we have in the model?

Reply:
What do you mean the model? It expects a k8 resource

Reply:
You're not adding version because it's not a real resource -- the model we use for the kind you're specifying has the data -- https://github.com/opendatahub-io/odh-dashboard/blob/main/frontend/src/api/models/openShift.ts#L17-L22

Reply:
So grab that from the constant to use? Even tho it doesn't get used

Reply:
This is what I did on my branch
```
export const convertBYONImageToK8sResource = (image: BYONImage): K8sResourceCommon => ({
  kind: 'ImageStream',
  apiVersion: 'image.openshift.io/v1',
  metadata: {
    name: image.name,
    annotations: {
      'openshift.io/display-name': image.display_name,
    },
  },
});
```

Reply:
@Gkrumbach07 How is the next developer going to know you chose to put an empty string in a field that is required that we chose to use later in a generic component? If you use nothing, don't adjust the types to say it's optional, and just say "it's not used today" -- TS can't even help us later when we might choose to support the versions. You can hope we remember this, hope we check every reference (this is a highly used component, with this one edge case from legacy code), or just in general hope that when we use it later it supports being empty despite being required in the type.

We need to be mindful of the code we write "tomorrow" and not make our current code fail because we just chose to, otherwise we'll always be chasing bugs. Juntao's solution works fine and has some future proof mentality to it.